lib/screens/v2/candidate_generator/dup/departures.ex

```diff
@@ -460,7 +461,7 @@ defmodule Screens.V2.CandidateGenerator.Dup.Departures do
 
   # If we are currently overnight, returns the first schedule of the day for each route_id and direction for each stop.
```

Comment:
This summary of the logic could use an update, as could the comments on the various clauses of `get_next_schedule_for_route`.